regression/test.pl

```diff
@@ -393,9 +393,9 @@ ($)
 print "\n";
 
 if($failures == 0) {
-  print "All tests were successful";
+  print "\e[92mAll tests were successful\e[0m";
 } else {
-  print "Tests failed\n";
+  print "\e[91mTests failed\e[0m\n";
```

Comment:
I'm not sure 91/92 are widely supported? I'd try with 31/32, maybe that works on Travis as well.

Reply:
I have just pushed a commit to try this out.